classes/LocalizationPack.php

```diff
@@ -369,6 +351,9 @@ protected function _installCurrencies($xml, $install_mode = false)
 
                         return false;
                     }
+                    Cache::clear();
+                    $this->setCurrencyCldrData($currency, $defaultLang);
```

Comment:
Something I'm not sure I understand here, currencies are not installed via CLDR? seems like they are installed by two sources which forces to get CLDR info here, would it be possible that only CLDR installs them and the the legacy Currency class calls it when it fetches its data? even in constructor?

Reply:
You suggestion is actually the old way. The objective from now is to install the default localization data at a currency creation, then allow the merchant to modify these values afterwards.

The old way did not allow this, as it was always overriding the values from the database.

Reply:
Oooohh, I see the interest! Thanks